package.json

```diff
@@ -85,6 +84,7 @@
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1",
+    "promise-polyfill": "^8.2.0",
```

Comment:
1) now matches `@ember/test-helpers` polyfill
2) works around ie11 + es6-promise + babel + webpack "glitch" that causes some fun to debug issues.

Reply:
👍